x-pack/plugins/observability_solution/infra/public/pages/logs/page_content.tsx

```diff
@@ -81,7 +81,7 @@ export const LogsPageContent: React.FunctionComponent = () => {
                 </EuiHeaderLink>
                 <LazyAlertDropdownWrapper />
                 <EuiHeaderLink
-                  href={getUrlForApp('/observabilityOnboarding')}
+                  href={getUrlForApp('/observabilityOnboarding', { path: '/?category=logs' })}
```

Comment:
**nit:** would be great to replace this URLs with the locator usage to standardize navigation to the onboarding page 👌